var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -70,7 +70,7 @@ class Qt(Package):
     depends_on("zlib")
     depends_on("dbus", when='@4:+dbus')
     depends_on("libtiff")
-    depends_on("libpng@1.2.56", when='@3')
+    depends_on("libpng@1.2.57", when='@3')
```

Comment:
No.  But I surmised from the `libpng` website announcement that they bumped up the minor version number by one on all their "latest" versions to indicate that the security problem had been fixed.  It's an educated guess ;-)